QUEST_LV_0100.tsv

```diff
@@ -2099,16 +2099,16 @@ QUEST_LV_0100_20150317_002098	Do not forget to use stimulants every time you run
 QUEST_LV_0100_20150317_002099	$If you feel you are in danger, quickly run away.{nl}The place is more dangerous due to the vicious atmosphere.
 QUEST_LV_0100_20150317_002100	{memo X}
 QUEST_LV_0100_20150317_002101	$I am not afraid of dying.{nl}The Goddess Saule will guide me well.
-QUEST_LV_0100_20150317_002102	Echart is the faithful protector of the Mausoleum.{nl}But, once it gets ruined, it never comes back to it's previous status.
-QUEST_LV_0100_20150317_002103	The broken epitaph of the Mausoleum
-QUEST_LV_0100_20150317_002104	The protector, Shnayim.{nl}Defeat the intruder who disturbs Zachariel's rest.
-QUEST_LV_0100_20150317_002105	The epitaph of the Mausoleum
-QUEST_LV_0100_20150317_002106	The protector will be disturbance to you when the maigic that flows in the Mausoleum get disrupted.
+QUEST_LV_0100_20150317_002102	$Achat is the faithful protector of the Mausoleum.{nl}But, once it gets corrupted, it will never return to its original state.
+QUEST_LV_0100_20150317_002103	$Broken Mausoleum Epitaph
+QUEST_LV_0100_20150317_002104	$Protector Shnayim.{nl}Defeat the intruder who disturbs Zachariel's rest.
+QUEST_LV_0100_20150317_002105	$Mausoleum Epitaph
```

Comment:
Mausoleum -> Royal Mausoleum
Epitaph -> Inscription ? (I've explained this in a later note, please check that.)